pages/index.tsx

```diff
@@ -38,94 +47,115 @@ export async function getStaticProps() {
     storage: ZeitgeistIpfs(),
   });
 
-  const banners = await getBanners();
+  const news = await getNews();
 
   const [
     featuredMarkets,
     trendingMarkets,
+    bannerPlaceholder,
     categoryPlaceholders,
-    bannerPlaceHolders,
+    newsImagePlaceholders,
     categoryCounts,
     stats,
+    ztgHistory,
+    chainProperties,
   ] = await Promise.all([
     getFeaturedMarkets(client, sdk),
     getTrendingMarkets(client, sdk),
+    getPlaiceholder(`/banner.png`),
     getPlaiceholders(
       CATEGORIES.map((cat) => `${cat.imagePath}`),
       { dir: `${path.join(process.cwd())}/public/` },
     ),
     getPlaiceholders(
-      banners.map((slide) => slide.imageUrl ?? ""),
+      news.map((slide) => slide.imageUrl ?? ""),
       { size: 16 },
     ),
     getCategoryCounts(
       client,
       CATEGORIES.map((cat) => cat.name),
     ),
     getNetworkStats(sdk),
+    getZTGHistory(),
+    sdk.api.rpc.system.properties(),
   ]);
 
   return {
     props: {
-      banners: banners,
+      news: news,
       featuredMarkets: featuredMarkets ?? [],
       trendingMarkets: trendingMarkets ?? [],
+      bannerPlaceholder: bannerPlaceholder.base64 ?? "",
       categoryCounts: categoryCounts,
       categoryPlaceholders: categoryPlaceholders.map((c) => c.base64) ?? [],
-      bannerPlaceHolders: bannerPlaceHolders.map((c) => c.base64) ?? [],
+      newsImagePlaceholders: newsImagePlaceholders.map((c) => c.base64) ?? [],
       stats,
+      ztgHistory,
+      chainProperties: chainProperties.toHuman(),
```

Comment:
`toHuman` can do weird things like adding commas to big numbers. Might just be better to pull out what you need, it will be less data send to the client too

Reply:
I find `toPrimitive` is often a better option if you wanted to go down that route

Reply:
Should also be fixed